ci-operator/config/openshift/kubevirt-csi-driver/openshift-kubevirt-csi-driver-master.yaml

```diff
@@ -87,67 +88,15 @@ tests:
   commands: make test
   container:
     from: src
-- as: cluster-api-provider-kubevirt-with-hypershift
+- as: e2e-kubevirt-aws-csi
   steps:
-    cluster_profile: aws-virtualization
+    cluster_profile: aws
     env:
-      BASE_DOMAIN: cnv-ci.syseng.devcluster.openshift.com
+      CNV_PRERELEASE_VERSION: "4.13"
```

Comment:
I had that in earlier versions, and the install would fail because the aws-virtualization profile didn't have that particular domain. So I switched to the aws profile.

Reply:
We need to solve it. We *must* use the cnv acount.

Reply:
Do you mean the "cnv-ci.syseng.devcluster.openshift.com" domain? It's in use with the cnv account in many tests.

Reply:
yeah, I can change it again, so I can show you what I am seeing.

Reply:
Okay there you go ```level=fatal msg=failed to fetch Common Manifests: failed to fetch dependency of "Common Manifests": failed to generate asset "DNS Config": getting public zone for "cnv-ci.syseng.devcluster.openshift.com": No public route53 zone found matching name "cnv-ci.syseng.devcluster.openshift.com"```

Reply:
i'm not sure if it is related, but you need to run `make jobs`

Reply:
apparently it is related. I did a make jobs locally and it appears to be progressing to the point of installing the different components, the jobs lane failed because I haven't rebased on the latest.